src/components/passphrase/passphrase.js

```diff
@@ -25,11 +25,12 @@ app.directive('passphrase', ($rootScope, $document, Passphrase, dialog, $mdMedia
      */
     const generateAndDoubleCheck = (seed) => {
       const passphrase = Passphrase.generatePassPhrase(seed);
-      const label = 'Save';
+      const label = attrs.okButtonLabel || 'Save';
```

Comment:
Please remove fixings related to save button title. I had to implement it in #359